distribution/lib/Standard/Base/0.0.0-dev/src/Data/Ordering.enso

```diff
@@ -20,6 +188,14 @@ type Ordering
     ## A representation that the first value orders as greater than the second.
     Greater
 
+    ## Compares to values and returns an Ordering
+    compare : Any -> Any -> Ordering ! (Incomparable_Values | Type_Error)
```

Comment:
It is surprising to see `compare` method here:
- I'd expect one to use `<`, `==`, etc.
- at other places in the PR we are *removing* `compare_to` method
- shall the name of this method (if it has to stay) be `compare_to` then?

Reply:
This is just a helper static method. This is essentially a replacement for call-sites that needed `Any.compare_to`.